src/PriceCalculatorKata/Product.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace PriceCalculatorKata
+{
+    class Product
+    {
+        public string Name { get; set; }
+        public int UPC { get; set; }
+        private double _price;
+        public double Price
+        {
+            get { return _price; }
+            set { _price = Math.Round(value, 2); }
```

Comment:
Product is a model class that would hold the data. So such calculations better be moved to a separate class (utility/extension).

Reply:
and this class should only have:
public string Name {get;set;}
public int UPC {get;set;}
public double Price {get;set;}
....